contracts/src/v0.8/dev/ocr2dr/OCR2DRClientExample.sol

```diff
@@ -0,0 +1,50 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "./OCR2DRClient.sol";
+import "./OCR2DR.sol";
+import "../../ConfirmedOwner.sol";
+
+/**
+ * @title OCR2DR example client contract implementation
+ */
+contract OCR2DRClientExample is OCR2DRClient, ConfirmedOwner {
+  bytes32 public lastRequestId;
+  bytes public lastResponse;
+  bytes public lastError;
+
+  constructor(address oracle) OCR2DRClient(oracle) ConfirmedOwner(msg.sender) {}
+
+  function SendRequest(
+    string calldata source,
+    bytes calldata secrets,
+    string[] calldata args,
+    uint256 subscriptionId
+  ) external onlyOwner returns (bytes32) {
+    OCR2DR.Request memory req;
+    req.source = source;
+    req.secrets = secrets;
+    req.args = args;
+    req.codeLocation = OCR2DR.Location.Inline;
+    req.secretsLocation = OCR2DR.Location.Inline;
+    req.language = OCR2DR.CodeLanguage.JavaScript;
+    return sendRequest(req, subscriptionId);
```

Comment:
A hypothetic external application will not be able to get the generated `requestId`.
Instead of returning a `requestId`, I suggest to store it in `lastRequestId` and then compare in `fulfillRequest`, this will show a good pattern of `requestId` verification.

Reply:
Not sure how complete this example is supposed to be, but I'm guessing they'd probably be emitting a log event too, at the end of these functions.

Reply:
Events are already emitted from OCR2DRClient that this contract inherits from. I don't think we need separate ones.